webauthn/helpers/structs.py

```diff
@@ -85,13 +85,11 @@ def _pydantic_v2_validate_bytes_fields(
             """
             field = cls.model_fields[info.field_name]  # type: ignore[attr-defined]
 
-            if field.annotation != bytes:
+            # UserHandle is defined as Optional[bytes]
```

Comment:
Can you add to this comment making it more explicit that `Optional[bytes]` won't annotate as a `bytes` field hence the need for this field-specific check?